cmdeploy/src/cmdeploy/dovecot/expunge.cron.j2

```diff
@@ -8,3 +8,4 @@
 # or only temporary (but then they shouldn't be around after {{ config.delete_mails_after }} days anyway).
 2 0 * * * vmail find /home/vmail/mail/{{ config.mail_domain }} -path '*/tmp/*' -mtime +{{ config.delete_mails_after }} -type f -delete
 2 0 * * * vmail find /home/vmail/mail/{{ config.mail_domain }} -path '*/.*/tmp/*' -mtime +{{ config.delete_mails_after }} -type f -delete
+3 0 * * * vmail find /home/vmail/mail/{{ config.mail_domain }} -name 'maildirsize' -type f -delete
```

Comment:
alex wrote in the issue: 
> Another thing that I just tested is simply removing maildirsize file, it worked and resulted in quota update. It is not nice because the quota has to be recalculated, but if we only do it when removing messages, may be fine.

It's probably not a big deal but i think writing a python script (maybe chatmaild.expunge_messages) and only removing maildirsize if files were actually deleted should be easy enough and also goes a bit of the way of making this per-user configurable maybe for 1.46.